buildSrc/subprojects/packaging/src/main/kotlin/gradlebuild/shaded-jar.gradle.kts

```diff
@@ -95,13 +95,14 @@ fun createConfigurationToShade() = configurations.create("jarsToShade") {
         isCanBeConsumed = false
         withDependencies {
             this.add(project.dependencies.create(project))
-            this.add(project.dependencies.create(project.dependencies.platform(project(":distributionsDependencies"))))
+            this.add(project.dependencies.create(project.dependencies.platform(project(":distributions-dependencies"))))
         }
     }
 
 fun addShadedJarTask(shadedJarExtension: ShadedJarExtension): TaskProvider<ShadedJar> {
     val configurationToShade = shadedJarExtension.shadedConfiguration
 
+    // TODO (donat) convert project name to kebab-case before prepending it to the task name
```

Comment:
Would be nice if these TODOs can be addressed directly (this PR, or a direct follow up). :)